### PR TITLE
update

### PR DESCRIPTION
**Pull request by Yizhuzhu**
1. Add Therapy component's CSS.
2. Return drugs in order when adding therapy.

### CODE REVIEW

File: web/yo/app/scripts/directives/addtherapy.js [hunk 1]
```diff
@@ -77,10 +77,15 @@ angular.module('oncokbApp')
 
                 $scope.loadDrugs = function($query) {
                     $scope.addTherapyError = false;
-                    return drugs.filter(function (drug) {
-                        var lowerCaseQuery = $query.toLowerCase();
-                        return drug.drugName.toLowerCase().indexOf(lowerCaseQuery) != -1 || (!drug.synonyms ? false : drug.synonyms.join(',').toLowerCase().indexOf(lowerCaseQuery) != -1);
-                    })
+                    var lowerCaseQuery = $query.toLowerCase();
+                    var result = [];
+                    result = drugs.filter(function (drug){
```
Comment (zhx828) [inline web/yo/app/scripts/directives/addtherapy.js]: Why twice?

Comment (Yizhuzhu) [reply web/yo/app/scripts/directives/addtherapy.js]: First: return drugs whose drug name contain the query.
Second: return drugs whose synonyms contain the query.
To make drug list in order.

Comment (zhx828) [reply web/yo/app/scripts/directives/addtherapy.js]: I would instead considering just introduce a sorting function. `.filter` is not for this purpose, and you also need to do a uniq and concat. Please also consider then a drug name starts with the query should have a higher priority then the drug name has the query.

File: web/yo/app/scripts/directives/addtherapy.js [hunk 2]
```diff
@@ -97,6 +98,36 @@ angular.module('oncokbApp')
                     addTherapy(index);
                 };
 
+                function getDrugsInOrder(lowerCaseQuery) {
+                    _.forEach(drugs, function(drug){
```
Comment (zhx828) [inline web/yo/app/scripts/directives/addtherapy.js]: It's minor, but I would rather use _.map to assign the weight. It's more elegant than constantly returning.

File: web/yo/app/scripts/directives/addtherapy.js [hunk 3]
```diff
@@ -77,10 +77,11 @@ angular.module('oncokbApp')
 
                 $scope.loadDrugs = function($query) {
                     $scope.addTherapyError = false;
+                    var lowerCaseQuery = $query.toLowerCase();
+                    getDrugsInOrder(lowerCaseQuery);
                     return drugs.filter(function (drug) {
```
Comment (zhx828) [inline web/yo/app/scripts/directives/addtherapy.js]: And I would rather do a filter then a sort to make it more efficient. Also you don't really need to sign a weight on items you don't want to include.

File: web/yo/app/scripts/directives/addtherapy.js [hunk 4]
```diff
@@ -97,6 +98,36 @@ angular.module('oncokbApp')
                     addTherapy(index);
                 };
 
+                function getDrugsInOrder(lowerCaseQuery) {
+                    _.forEach(drugs, function(drug){
+                        if(drug.drugName.toLowerCase()===lowerCaseQuery){
```
Comment (zhx828) [inline web/yo/app/scripts/directives/addtherapy.js]: drugName should be lowercased  at the top to avoid repeat converting

**Pull request merged**